.github/CODEOWNERS

```diff
@@ -55,32 +61,46 @@ src/applications/disability-benefits @department-of-veterans-affairs/benefits-te
 src/applications/claims-status @department-of-veterans-affairs/benefits-team-1-frontend
 src/applications/appeals @department-of-veterans-affairs/benefits-team-1-frontend @department-of-veterans-affairs/vsa-authd-exp-frontend
 src/applications/sah @department-of-veterans-affairs/benefits-team-1-frontend
+src/applications/static-pages/coe-access @department-of-veterans-affairs/benefits-team-1-frontend
```

Comment:
I think `benefits-non-disability` owns COE now

Reply:
Got it - thanks!